Sources/MongoSwift/MongoClient.swift

```diff
@@ -278,6 +278,96 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+    /**
+     * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+     * system collections i.g. "config", "local", and "admin" databases. By default, the type `CollectionType` is
+     * associated with the `fullDocument` field in the `ChangeStreamsDocument`.
+     * - Parameters:
+     *   - Pipeline: The pipeline of stages to append to an initial `ChangeStream` stage.
+     *   - Options: An optional `ChangeStreamOptions` to use on the initial `ChangeStream` stage.
+     *   - Session: An optional `ChangeStream` to use with this change stream.
+     * - Returns: a `ChangeStream` on all collections in all databases in a cluster.
+     * - Throws:
+     *   - `ServerError.commandError` if an error occurs on the server while creating the cursor.
+     *   - `UserError.invalidArgumentError` if the options passed formed an invalid combination or the pipeline passed
+     *     is invalid.
+     *   - `UserError.invalidArgumentError` if the `_id` field is projected out of the change stream documents by the
+     *     pipeline.
+     * - SeeAlso: https://docs.mongodb.com/manual/reference/system-collections/
+     */
+     public func watch(_  pipeline: [Document],
```

Comment:
I think in all these methods, pipeline should have a default value of `[:]` (empty document), so that a user can omit it if they don't want to do anything fancy.

Reply:
Updated. I set its default value to [] though, then just wrap it in a `Document` before sending it to libmongoc, similar to how it's done [here](https://github.com/mongodb/mongo-swift-driver/blob/00fe70e9764d3d9a01a3cbd409470a7ab11db390/Sources/MongoSwift/MongoCollection%2BRead.swift#L59).

Reply:
oh yes, i was thinking it's a document, but it's an array 👍